flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java

```diff
@@ -278,15 +278,15 @@
 	 * Unlimited be default.
 	 */
 	public static ConfigOption<Long> rateLimitingInputBytesPerSeconds(String scheme) {
-		return ConfigOptions.key("fs." + scheme + ".limit.rateLimitingInput").defaultValue(0L);
+		return ConfigOptions.key("fs." + scheme + ".limit.input-rate-limit").defaultValue(0L);
```

Comment:
Please rename also variables/methods to match the new config name and match the other properties above:
`s/rateLimitingInputBytesPerSeconds/fileSystemConnectionLimitInputRate/g`
`s/rateLimitingOututBytesPerSeconds/fileSystemConnectionLimitOutputRate/g`